src/SFA.DAS.ApplyService.Web/Views/OrganisationSearch/Results.cshtml

```diff
@@ -55,7 +55,7 @@
                         {
                             <tr class="govuk-table__row govuk-table__row--no-top-padding-on-first-child">
                                 <td class="govuk-table__cell govuk-!-padding-bottom-4">
-                                    <form method="post" asp-action="@(result.OrganisationType != null ? "Confirm" : "Type")" asp-controller="OrganisationSearch" novalidate>
+                                    <form method="post" asp-action="@(result.RoEPAOApproved ? "Confirm" : "Type")" asp-controller="OrganisationSearch" novalidate>
```

Comment:
Can this c# be pushed into the VM?

Reply:
Will be quite difficult to do as this VM is shared across the Preamble journey

Reply:
Surely a new OrganisationSearchAction property with that code in wouldn't hurt anywhere else?

Reply:
So it turns out it is an internal API type rather than a VM. @davidgouge are you happy to approve if we resolve this issue another time?